compiler/AST/expr.cpp

```diff
@@ -3722,9 +3722,67 @@ CallExpr::insertAtTail(BaseAST* ast) {
 }
 
 
-FnSymbol* CallExpr::isResolved(void) {
-  SymExpr* base = toSymExpr(baseExpr);
-  return base ? toFnSymbol(base->var) : NULL;
+bool CallExpr::isEmpty() const {
+  return primitive == NULL && baseExpr == NULL;
+}
+
+
+// MDN 2016/01/29: This will become a predicate
+FnSymbol* CallExpr::isResolved() const {
```

Comment:
To me, this function should really return a boolean and calls to isResolved() that want the function it resolves to should just call resolvedFunction() instead.  But that'd likely touch a larger swathe of code so I understand why you didn't.  Still seems unfortunate